modules/admin/component-protocol/components/personal-workbench/messageList/approval.go

```diff
@@ -39,6 +41,17 @@ func (l *MessageList) doFilterApproval() (data *list.Data) {
 		UserIDs: append(blockUsers, deployUsers...),
 	}
 
+	sort.Slice(data.List, func(i, j int) bool {
```

Comment:
it's wrong if you have more than 1000 db records.
you should implement in db side.

Reply:
Yes, we may union tables or refactor approval function. 
it works as there are less than 300 records in db.

Reply:
then, at least leave some TODO